tensorflow/core/kernels/data/dataset_test_base.h

```diff
@@ -47,6 +47,10 @@ class DatasetOpsTestBase : public ::testing::Test {
 
   ~DatasetOpsTestBase() {}
 
+  // Returns Status::OK() if the input two tensors have the same shape, dtype,
```

Comment:
This is incorrect. The function can return `Status::OK()` even if the expectation fails. I would simply say here that the method validates whether the two tensors have the same shape, dtype, and value.